lib/galaxy/webapps/galaxy/api/configuration.py

```diff
@@ -70,7 +70,9 @@ def index(
         Pass in `view` and a comma-seperated list of keys to control which
         configuration settings are returned.
         """
-        return _index(self.configuration_manager, trans, view, keys)
+        index = _index(self.configuration_manager, trans, view, keys)
+        index["asgi_enabled"] = True
+        return index
```

Comment:
```suggestion
        return _index(self.configuration_manager, trans, view, keys)
```
Should we just dispatch on the major version ? Starting from 22.05 every server is asgi_enabled

Reply:
I can't find this being used anyway. This was left over from the older style controllers.